packages/commonwealth/server/migrations/20230503232942-add-thread-id-to-notif.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.sequelize.query(`
+        CREATE TEMPORARY TABLE notif_to_delete AS
+        SELECT id
+        FROM "Notifications"
+        WHERE notification_data::jsonb ? 'thread_id' AND
+                (notification_data::jsonb ->> 'thread_id') NOT IN (SELECT id::text FROM "Threads");
```

Comment:
Thanks again @timolegros did it work?
if it worked, seems like we are ready! 

- Can we check once all notification without thread_id prop are not deleted? I can see you added check, just double check

- How much time it took to run this migration? if you know!

Reply:
https://dashboard.heroku.com/apps/commonwealth-staging/activity/releases/8b796e39-c4e3-4b5b-85ed-e3b0f2b919f6

== 20230503232942-add-thread-id-to-notif: migrated (12.984s)
which is great, thanks!

Reply:
We have both thread_ids null, not null
<img width="329" alt="image" src="https://user-images.githubusercontent.com/4791635/236590419-a8fef49b-685f-4821-b33c-a163ddd75a4c.png">

<img width="424" alt="image" src="https://user-images.githubusercontent.com/4791635/236590469-afc04b98-017e-4b4a-9ea7-e2ba5792e1ff.png">

- prod Notifications count is 194878
- staging Notifications count is 194673 

Notifications & NotificationRead count looks healthy even after migration thanks!